docs-developer/README.md

```diff
@@ -24,3 +24,4 @@ This project is a client for reading profiles from the Gecko Profiler and potent
  * [Potential performance data sources in Gecko](./data-sources.md)
  * [Call tree](./call-tree.md)
  * [Frames, funcs, stacks and CallNodes in C++](./call-nodes-in-cpp.md)
+ * [Deploy](./deploying.md)
```

Comment:
nit: can you please use the text `Deploying to production`?
thanks !

Reply:
Sorry doing the same in a min

Reply:
@julienw made the requested changes . I think ready for another review :)